organizations/models.go

```diff
@@ -170,34 +170,41 @@ var StatusExpiryTime = map[string]string{
 }
 
 type Status struct {
-	Tag        string `json:"tag" bson:"tag"`
-	Text       string `json:"text" bson:"text"`
-	ExpiryTime string `json:"expiry_time" bson:"expiry_time"`
+	Tag        		string 				`json:"tag" bson:"tag"`
+	Text       		string 				`json:"text" bson:"text"`
+	ExpiryTime 		string 				`json:"expiry_time" bson:"expiry_time"`
+}
+
+type StatusHistories struct {
+	TagHistory		string  `json:"tag_history" bson:"tag_history"`
+	TextHistory		string  `json:"text_history" bson:"text_history"`
+	ExpiryHistory	string	`json:"expiry_history" bson:"expiry_history"` 			
 }
 
 type Member struct {
-	ID          primitive.ObjectID `json:"_id" bson:"_id"`
-	OrgID       string             `json:"org_id" bson:"org_id"`
-	Files       []string           `json:"files" bson:"files"`
-	ImageURL    string             `json:"image_url" bson:"image_url"`
-	FirstName   string             `json:"first_name" bson:"first_name"`
-	LastName    string             `json:"last_name" bson:"last_name"`
-	Email       string             `json:"email" bson:"email"`
-	UserName    string             `bson:"user_name" json:"user_name"`
-	DisplayName string             `json:"display_name" bson:"display_name"`
-	Bio         string             `json:"bio" bson:"bio"`
-	Status      Status             `json:"status" bson:"status"`
-	Presence    string             `json:"presence" bson:"presence"`
-	Pronouns    string             `json:"pronouns" bson:"pronouns"`
-	Phone       string             `json:"phone" bson:"phone"`
-	TimeZone    string             `json:"time_zone" bson:"time_zone"`
-	Role        string             `json:"role" bson:"role"`
-	JoinedAt    time.Time          `json:"joined_at" bson:"joined_at"`
-	Settings    *Settings          `json:"settings" bson:"settings"`
-	Deleted     bool               `json:"deleted" bson:"deleted"`
-	DeletedAt   time.Time          `json:"deleted_at" bson:"deleted_at"`
-	Socials     []Social           `json:"socials" bson:"socials"`
-	Language    string             `json:"language" bson:"language"`
+	ID          	primitive.ObjectID `json:"_id" bson:"_id"`
+	OrgID       	string             `json:"org_id" bson:"org_id"`
+	Files       	[]string           `json:"files" bson:"files"`
+	ImageURL    	string             `json:"image_url" bson:"image_url"`
+	FirstName   	string             `json:"first_name" bson:"first_name"`
+	LastName    	string             `json:"last_name" bson:"last_name"`
+	Email       	string             `json:"email" bson:"email"`
+	UserName    	string             `bson:"user_name" json:"user_name"`
+	DisplayName 	string             `json:"display_name" bson:"display_name"`
+	Bio         	string             `json:"bio" bson:"bio"`
+	Status      	Status             `json:"status" bson:"status"`
+	StatusHistory	[]StatusHistories  `json:"status_history" bson:"status_history"`
```

Comment:
I was thinking of having the history embedded in the status struct. We can handle it as an array of Status history sef, Instead of having statushistory live alone like this.